frontend/src/metabase/query_builder/components/filters/modals/BulkFilterModal/utils.ts

```diff
@@ -0,0 +1,63 @@
+import StructuredQuery from "metabase-lib/lib/queries/StructuredQuery";
+import Filter from "metabase-lib/lib/queries/structured/Filter";
+
+// fix between filters with missing or misordered arguments
+export function fixBetweens(query: StructuredQuery): StructuredQuery {
+  const betweenFilters = query
+    .filters()
+    .filter(filter => filter.operatorName() === "between");
+
+  for (const filter of betweenFilters) {
+    const validArguments = countValidArumgents(filter);
+    if (validArguments === 1) {
+      return fixBetweens(handleEmptyBetween(filter));
+    } else if (validArguments === 2 && hasBackwardsArguments(filter)) {
+      return fixBetweens(swapFilterArguments(filter));
```

Comment:
Same as the comment above.
```suggestion
      query = fixBetweens(swapFilterArguments(filter));
```